cli/tests/bundle/fixture11.out

```diff
@@ -6,9 +6,9 @@ var O;
     O1[O1["A"] = 0] = "A";
     O1[O1["B"] = 1] = "B";
     O1[O1["C"] = 2] = "C";
-})((void 0) || (O = {
+})(O || (O = {
 }));
-const O1 = void 0;
+const O1 = O;
```

Comment:
@kitsonk please take a look - the result of this test changed after the upgrade, I'm not sure if that's ok

Reply:
Also CC @kdy1

Reply:
I think it's ok. Actually previous output is a bug.

Previously exported `O` was undefined, but it should be `O1`, which is enum.

Reply:
Yeah, it was a big but we didn't notice it.